Makefile

```diff
@@ -198,6 +201,7 @@ get-test-namespace: out/test-namespace
 .PHONY: deploy-e2e-crds
 deploy-e2e-crds:
 	$(Q)kubectl --namespace $(TEST_NAMESPACE) apply -f ./test/third-party-crds/postgresql_v1alpha1_database_crd.yaml
+	$(Q)kubectl --namespace $(TEST_NAMESPACE) apply -f ./test/third-party-crds/etcd_v1beta2_etcdcluster_crd.yaml
```

Comment:
`make deploy-e2e-crds` throws an error:

```
➜  service-binding-operator git:(skip-annotation) make deploy-e2e-crds 
kubectl --namespace  apply -f ./test/third-party-crds/postgresql_v1alpha1_database_crd.yaml
Error: unknown shorthand flag: 'f' in -f
See 'kubectl --help' for usage.
make: *** [Makefile:203: deploy-e2e-crds] Error 1
````

Reply:
Fixed in b3b5b397f4ca4840a3a11909dedb939c71659653.